src/cleanandgo/EquipmentAndSupplies.java

```diff
@@ -88,12 +88,17 @@ private static void printAnnualCleaningSuppliesExpenses() {
         try{
             // get connected to the DB
             Connection conn = DBConnection.getConnected();
-            String query = " NEED TO PUT QUERY FOR CLEANING SUPPLIES HERE...";
+            String query = "SELECT sum(amount_due) FROM Purchased_Cleaning_Supplies " + 
+                            "WHERE date_of_purchase BETWEEN DATE_ADD(now(), interval -12 month) and now(); ";
+                            /* Original query, which is better?
```

Comment:
Okay, I will leave the interval query as the final one. That is a very good idea to make the function smoother in other scenarios. Since it works as-is for now, I will leave it, and if I have time I will alter to get the year and 12 months in that interval.